packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -334,6 +334,149 @@ void main() {
     expect(tester.getBottomLeft(find.text('Title')).dy, 44.0 - 8.0); // Extension gone, (static part - padding) left.
   });
 
+  testWidgets(
+      'Border is displayed by default in sliver nav bar',
+      (WidgetTester tester) async {
+    final ScrollController scrollController = new ScrollController();
```

Comment:
since you're not using any of these in the test, you don't have to define any of these variables